src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/PasswordDeriveBytes.cs

```diff
@@ -50,7 +52,8 @@ public PasswordDeriveBytes(byte[] password, byte[] salt, string hashName, int it
             _cspParams = cspParams;
         }
 
-        public string HashName
+        [DisallowNull]
+        public string? HashName
```

Comment:
Since `hashName` is marked as non-nullable in the ctor, and we were going to put `[DisallowNull]` anyways, I'd just make the property be non-nullable.

null will make it surprisingly far, but is going to eventually blow up.